config/routes.rb

```diff
@@ -13,6 +13,8 @@
 
     get "/subscribers/:address/subscriptions", to: "subscribers#subscriptions",
       constraints: { address: /.+@.+\..+/ }
+    post "/subscribers/:address/change-address", to: "subscribers#change_address",
```

Comment:
I don't know how much we care about REST, but we could instead make this a `PATCH /subscribers/:address` endpoint and then the body will be replacement fields.

```
PATCH /subscribers/test@example.com

address=test2@example.com
```

Reply:
Yes good idea, done